src/components/viesti/new-ketju.svelte

```diff
@@ -78,6 +78,12 @@
       },
       response => {
         resources = Maybe.Some(response);
+        if (
+          !isAllowedToSendToEveryone(
+            R.prop('whoami', R.head(resources.toArray()))
```

Comment:
response.whoami

Reply:
1. maybe:tä ei käytetä näin
2. whoami on saatavilla suoraan responsesta niin ei sitä kannata yrittää kaivaa tuolta resources objektista